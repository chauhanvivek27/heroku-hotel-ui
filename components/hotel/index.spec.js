import React from "react";
import { mount } from "enzyme";
import { renderToJson } from "enzyme-to-json";
import Hotel from "./index";

describe("Header Render test", () => {
  let wrapper;

  it("should render correctly", () => {
    wrapper = mount(<Hotel />);
    expect(renderToJson(wrapper.render())).toMatchSnapshot();
  });
});
