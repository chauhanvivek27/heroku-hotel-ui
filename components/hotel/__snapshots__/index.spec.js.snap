// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Render test should render correctly 1`] = `
<div
  class="hotel-search"
>
  <div
    class="header"
  >
    <h1
      class="headertitle"
    >
      Hotel Search
    </h1>
  </div>
  <div
    class="hotel-search__divsection"
  >
    <div
      class="panel panel-default"
    >
      <div
        class="formErrors"
      >
        <ul />
      </div>
    </div>
    <div
      class="hotel-search__field-row"
    >
      <div
        class="input-field"
      >
        <input
          id="txtCity"
          placeholder="City"
          type="text"
          value=""
        />
        <label
          class="topLabel"
          for="txtCity"
        >
          City
        </label>
      </div>
      <div
        class="input-field"
      >
        <input
          id="txtName"
          placeholder="Hotel Name"
          type="text"
          value=""
        />
        <label
          class="topLabel"
          for="txtName"
        >
          Hotel Name
        </label>
      </div>
    </div>
    <div
      class="hotel-search__field-row"
    >
      <div
        class="input-field"
      >
        <input
          id="txtPriceRange"
          placeholder="Price Range"
          type="text"
          value=""
        />
        <label
          class="topLabel"
          for="txtPriceRange"
        >
          Price Range
        </label>
      </div>
      <div
        class="input-field"
      >
        <input
          id="txtDateRange"
          placeholder="Date Range"
          type="text"
          value=""
        />
        <label
          class="topLabel"
          for="txtDateRange"
        >
          Date Range
        </label>
      </div>
    </div>
    <div
      class="btndiv"
    >
      <input
        class="btnsumbit"
        type="submit"
        value="Search"
      />
    </div>
  </div>
  <div
    class="hotel-search__record-msg"
  />
</div>
`;
